src/public/modules/forms/services/form-feedback.client.factory.js

```diff
@@ -32,22 +32,6 @@ function FormFeedback($q, $http) {
   let resourceUrl = '/:formId/feedback'
   const feedbackAdminUrl = '/:formId/adminform/feedback'
   let feedbackService = {
-    postFeedback: function (params, body) {
```

Comment:
Can't delete this function altogether, since this function will call the real feedback URL if `body.isPreview === false`.

Two options I can see:
1. In this FeedbackService, if `body.isPreview === true`, we return `deferred.resolve(... )` immediately without performing a network call, and only call our non-preview feedback endpoint if it is not a preview.
2. Remove the isPreview check in this service, and perform the check elsewhere (lets say `feedback-form.client.component`). This service will ALWAYS call the real non-preview feedback endpoint.

Reply:
Opting for Option 1. Removed chunks in `src/public/modules/forms/services/form-feedback.client.factory.js` are moved into the *else-clause* for `if (body.isPreview)`